src/config.js

```diff
@@ -41,100 +42,100 @@ require.config({
-            "amcharts-gauge": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/gauge",
-            "amcharts-pie": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/pie",
-            "amcharts-radar": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/radar",
-            "amcharts-serial": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/serial",
-            "amcharts-xy": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/xy",
-            "amcharts-gantt": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/gantt",
-            "amcharts-plugins-responsive": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/plugins/responsive/responsive",
-            "amcharts-plugins-dataloader": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/plugins/dataloader",
-            "amcharts-images": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/images/",
+                "amcharts": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/amcharts",
+                "amcharts-funnel": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/funnel",
+                "amcharts-gauge": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/gauge",
+                "amcharts-pie": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/pie",
+                "amcharts-radar": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/radar",
+                "amcharts-serial": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/serial",
+                "amcharts-xy": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/xy",
+                "amcharts-gantt": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/gantt",
+                "amcharts-plugins-responsive": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/plugins/responsive/responsive",
+                "amcharts-plugins-dataloader": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/plugins/dataloader",
+                "amcharts-images": "//" + window.location.hostname + "/amcharts/amcharts3/3.18.0/amcharts/images/",
 
-            "simpleheat": "//" + window.location.hostname + "/mourner/simpleheat/v0.3.0/simpleheat",
-            "autoComplete": "//" + window.location.hostname + "/Pixabay/JavaScript-autoComplete/1.0.4/auto-complete",
+                "simpleheat": "//" + window.location.hostname + "/mourner/simpleheat/v0.3.0/simpleheat",
+                "autoComplete": "//" + window.location.hostname + "/Pixabay/JavaScript-autoComplete/1.0.4/auto-complete",
 
-            "src": "../src"
-        }
-    });
+                "src": "../src"
+            }
+        });
 }
```

Comment:
Try to avoid gratutious white space changes (or make them on a separate commit with no "code" changes",